cypress/integration/work-order/create/create.spec.js

```diff
@@ -1759,6 +1766,301 @@ describe('Raise repair form', () => {
       )
     })
   })
+
+  context('When a user removes a contact detail', () => {
```

Comment:
Can we break this in to multiple spec files, it's getting ridiculous....

Reply:
Sure. Despite my initial reluctance, I'm starting to like using Cucumber for Cypress. These tests are pretty difficult to manage